redemptions.go

```diff
@@ -4,40 +4,11 @@ import "encoding/xml"
 
 // Redemption holds redeemed coupons for an account or invoice.
```

Comment:
It also might be a good idea to add constants for the state since there are only two possible states. For example: 

```
const (
     RedemptionStateActive = "active"
     RedemptionStateInactive = "inactive" 
) 
```